Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -1,4 +1,4 @@
-#:  * `tests` [`-v`] [`--coverage`] [`--generic`] [`--no-compat`] [`--only=`<test_script>`:`<test_method>] [`--seed` <seed>] [`--trace`] [`--online`] [`--official-cmd-taps`]:
+#:  * `tests` [`-v`] [`--coverage`] [`--generic`] [`--no-compat`] [`--only=`<test_name>] [`--seed` <seed>] [`--online`] [`--official-cmd-taps`]:
```

Comment:
Wouldn't be surprised if that wasn't possible with parallel_rspec since it likes to treat files as being the lowest division of tests (though I haven't checked). Workaround is probably best.